src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+body {
+  margin: 0 auto;
+}
+
+.big__form {
+  position: relative;
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use * selector

Reply:
done